src/Compilers/CSharp/Test/Syntax/Parsing/StatementParsingTests.cs

```diff
@@ -2473,6 +2588,27 @@ public void TestUsingSpecialCase2()
             Assert.NotNull(us.Statement);
         }
 
+        [Fact]
+        public void TestUsingVarSpecialCase2()
+        {
+            var text = "using f ? x = a;";
+            var statement = this.ParseStatement(text);
+
+            Assert.NotNull(statement);
+            Assert.Equal(SyntaxKind.LocalDeclarationStatement, statement.Kind());
+            Assert.Equal(text, statement.ToString());
+            Assert.Equal(0, statement.Errors().Length);
+            
+            var us = (LocalDeclarationStatementSyntax)statement;
+            Assert.NotNull(us.UsingKeyword);
+            Assert.Equal(SyntaxKind.UsingKeyword, us.UsingKeyword.Kind());
+            Assert.NotNull(us.Declaration);
+            Assert.Equal("f ? x = a", us.Declaration.ToString());
+
+        }
```

Comment:
Nit: bunch of unnecessary empty lines here and elsewhere.

Also, for testing syntax, have you tried tests that use this format: `TernaryVersusDeclaration_05`?
Those will show and validate the structure of the parse tree. A small trick to help writing such tests is to uncomment `//#define PARSING_TESTS_DUMP` in the base file (`ParsingTests`) so that the test output gives you the expected code.